stl/inc/type_traits

```diff
@@ -1840,7 +1840,11 @@ template <class _Ty, class _Uty>
 struct _Refwrap_has_ctor_from<_Ty, _Uty, void_t<decltype(_Refwrap_ctor_fun<_Ty>(_STD declval<_Uty>()))>> : true_type {};
 
 template <class _Ty>
-class reference_wrapper : public _Weak_types<_Ty>::type { // stand-in for an assignable reference
+class reference_wrapper
+#if !_HAS_CXX20

```

Comment:
There doesn't seem to be a big reason to condition this change on C++20. @StephanTLavavej do you know why this _Weak_types base was used at all? I don't see the members `_Is_f_or_pf` and/or `_Is_pmf` getting used, and `type` is shadowed.

Reply:
Oh derp, type isn't shadowed, it's the actual base. Don't mind me...